### ISSUE TITLE
Orpheus - Invalid & Unparsable DateTime Returned  

### ISSUE THREAD

**Issue by jmerhar** [Prowlarr/Prowlarr#773]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Prowlarr keeps sending health check failure notifications about Orpheus (to Telegram) despite the fact that the indexer works fine when searching, as well as when I click "Test". The issue seems to be due to failing to convert a datetime to an integer in the response.

### Expected Behavior

Automatic health checks should succeed for this indexer, and notifications should stop.

### Environment

```markdown
- OS: Ubuntu 18.04
- Prowlarr: 0.1.9.1313
- Docker Install: No
- Using Reverse Proxy: Yes
```

### What branch are you running?

Develop

### Trace Logs?

2022-01-11 12:25:57.4|Error|Orpheus|An error occurred while processing indexer feed. https://orpheus.network/ajax.php?action=browse&order_by=time&order_way=desc&filter_cat[3]=1&filter_cat[4]=1&filter_cat[7]=1

[v0.1.9.1313] Newtonsoft.Json.JsonSerializationException: Error converting value "2022-01-06 17:11:48" to type 'System.Int64'. Path 'response.results[0].groupTime', line 1, position 392.
 ---> System.FormatException: Input string was not in a correct format.
   at System.Number.ThrowOverflowOrFormatException(ParsingStatus status, TypeCode type)
   at System.String.System.IConvertible.ToInt64(IFormatProvider provider)
   at System.Convert.ChangeType(Object value, Type conversionType, IFormatProvider provider)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.EnsureType(JsonReader reader, Object value, CultureInfo culture, JsonContract contract, Type targetType)
   --- End of inner exception stack trace ---
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.EnsureType(JsonReader reader, Object value, CultureInfo culture, JsonContract contract, Type targetType)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.SetPropertyValue(JsonProperty property, JsonConverter propertyConverter, JsonContainerContract containerContract, JsonProperty containerProperty, JsonReader reader, Object target)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateList(IList list, JsonReader reader, JsonArrayContract contract, JsonProperty containerProperty, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateList(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, Object existingValue, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.SetPropertyValue(JsonProperty property, JsonConverter propertyConverter, JsonContainerContract containerContract, JsonProperty containerProperty, JsonReader reader, Object target)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.SetPropertyValue(JsonProperty property, JsonConverter propertyConverter, JsonContainerContract containerContract, JsonProperty containerProperty, JsonReader reader, Object target)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   at NzbDrone.Common.Serializer.Json.Deserialize[T](String json) in D:\a\1\s\src\NzbDrone.Common\Serializer\Newtonsoft.Json\Json.cs:line 48
   at NzbDrone.Common.Http.HttpResponse`1..ctor(HttpResponse response) in D:\a\1\s\src\NzbDrone.Common\Http\HttpResponse.cs:line 123
   at NzbDrone.Core.Indexers.Gazelle.GazelleParser.ParseResponse(IndexerResponse indexerResponse) in D:\a\1\s\src\NzbDrone.Core\Indexers\Definitions\Gazelle\GazelleParser.cs:line 27
   at NzbDrone.Core.Indexers.HttpIndexerBase`1.FetchPage(IndexerRequest request, IParseIndexerResponse parser) in D:\a\1\s\src\NzbDrone.Core\Indexers\HttpIndexerBase.cs:line 320
   at NzbDrone.Core.Indexers.HttpIndexerBase`1.FetchReleases(Func`2 pageableRequestChainSelector, Boolean isRecent) in D:\a\1\s\src\NzbDrone.Core\Indexers\HttpIndexerBase.cs:line 174
RequestUri: https://orpheus.network/ajax.php?action=browse&order_by=time&order_way=desc&filter_cat[3]=1&filter_cat[4]=1&filter_cat[7]=1;StatusCode: OK;ContentType: application/json; charset=utf-8;ContentLength: 20488;ContentSample: {"status":"success","response":{"currentPage":1,"pages":20,"results":[{"groupId":814117,"groupName":"Hugh Burns - Hal Leonard Flamenco Guitar Method: Learn to Play Flamenco Guitar with Step-by-Step Lessons and Authentic Pieces to Study and Play","torrentId":1770746,"tags":["flamenco.guitar","guitar","music","non.fiction"],"category":"E-Books","fileCount":1,"groupTime":"2022-01-06 17:11:48","size":7108740,"snatches":0,"seeders":0,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":814114,"groupName":"Andy Schneider - Bass Guitar Theory Nuts & Bolts","torrentId":1770739,"tags":["bass.guitar","non.fiction","sheet.music"],"category":"E-Books","fileCount":1,"groupTime":"2022-01-06 17:04:50","size":19093069,"snatches":0,"seeders":0,"leechers":1,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":814113,"groupName":"Richard Taruskin - The Oxford History of Western Music, Volumes 1-5","torrentId":1770738,"tags":["history","music","non.fiction"],"category":"E-Books","fileCount":1,"groupTime":"2022-01-06 17:00:48","size":95624132,"snatches":0,"seeders":0,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":805471,"groupName":"CECIL TAYLOR IN BERLIN '88","torrentId":1753986,"tags":["jazz"],"category":"E-Books","fileCount":1,"groupTime":"2021-12-18 03:09:56","size":73259155,"snatches":2,"seeders":3,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":804518,"groupName":"Peter Ames Carlin - Paul McCartney: A Life","torrentId":1752114,"tags":["biography","m4b","non.fiction","paul.mccartney"],"category":"Audiobooks","fileCount":1,"groupTime":"2021-12-15 19:57:11","size":532782630,"snatches":0,"seeders":0,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":802918,"groupName":"American Splendor [1976] (#1-17)","torrentId":1748554,"tags":["underground"],"category":"Comics","fileCount":17,"groupTime":"2021-12-12 06:14:24","size":541549052,"snatches":0,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":800315,"groupName":"Computer Music, Issue 303 (January 2022)","torrentId":1741649,"tags":["audio","electronicmusic","magazine","music.reviews","non.fiction","pdf","reviews","sound.music"],"category":"E-Books","fileCount":1,"groupTime":"2021-12-05 20:31:23","size":23884760,"snatches":1,"seeders":2,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":800314,"groupName":"Computer Music, Issue 302 (December 2021)","torrentId":1741648,"tags":["audio","electronicmusic","magazine","music.reviews","non.fiction","pdf","reviews","sound.music"],"category":"E-Books","fileCount":1,"groupTime":"2021-12-05 20:31:21","size":22037264,"snatches":0,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":800313,"groupName":"Computer Music, Issue 301 (November 2021)","torrentId":1741647,"tags":["audio","electronicmusic","magazine","music.reviews","non.fiction","pdf","reviews","sound.music"],"category":"E-Books","fileCount":1,"groupTime":"2021-12-05 20:31:18","size":22852259,"snatches":0,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":800312,"groupName":"Computer Music: Issue 300 Special (Autumn 2021)","torrentId":1741646,"tags":["audio","electronicmusic","magazine","music.reviews","non.fiction","pdf","reviews","sound.music"],"category":"E-Books","fileCount":1,"groupTime":"2021-12-05 20:31:15","size":22917824,"snatches":0,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":800295,"groupName":"Electronic Musician (January 2022)","torrentId":1741620,"tags":["audio","electronicmusic","magazine","music.reviews","non.fiction","pdf","reviews","sound.music"],"category":"E-Books","fileCount":1,"groupTime":"2021-12-05 20:04:07","size":14491683,"snatches":0,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":800294,"groupName":"Electronic Musician (December 2021)","torrentId":1741619,"tags":["audio","electronicmusic","magazine","music.reviews","non.fiction","pdf","reviews","sound.music"],"category":"E-Books","fileCount":1,"groupTime":"2021-12-05 20:04:05","size":14470480,"snatches":0,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":800293,"groupName":"Electronic Musician (November 2021)","torrentId":1741618,"tags":["audio","electronicmusic","magazine","music.reviews","non.fiction","pdf","reviews","sound.music"],"category":"E-Books","fileCount":1,"groupTime":"2021-12-05 20:04:03","size":14500510,"snatches":0,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":800288,"groupName":"Future Music, Issue 377 (16 November 2021)","torrentId":1741607,"tags":["audio","electronicmusic","magazine","music.reviews","non.fiction","pdf","reviews","sound.music"],"category":"E-Books","fileCount":1,"groupTime":"2021-12-05 19:50:51","size":21893941,"snatches":0,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":800287,"groupName":"Future Music, Issue 376 (19 October 2021)","torrentId":1741606,"tags":["audio","electronicmusic","magazine","music.reviews","non.fiction","pdf","reviews","sound.music"],"category":"E-Books","fileCount":1,"groupTime":"2021-12-05 19:50:48","size":22348628,"snatches":0,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":800286,"groupName":"Future Music, Issue 375 (21 September 2021)","torrentId":1741605,"tags":["audio","electronicmusic","magazine","music.reviews","non.fiction","pdf","reviews","sound.music"],"category":"E-Books","fileCount":1,"groupTime":"2021-12-05 19:50:44","size":21861500,"snatches":0,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":799070,"groupName":"The Absolute Sound (December 2020)","torrentId":1738353,"tags":["audio","classical","hifi","jazz","magazine","music.reviews","non.fiction","pdf","reviews","sound.music"],"category":"E-Books","fileCount":1,"groupTime":"2021-12-03 00:56:25","size":97475554,"snatches":0,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":798723,"groupName":"Ingo Neumayer - Die Geschichte der Donots: Heute Pl\u00e4ne, morgen Konfetti","torrentId":1736925,"tags":["biography","german","music","punk","rock"],"category":"Audiobooks","fileCount":159,"groupTime":"2021-12-02 01:45:15","size":1499682890,"snatches":1,"seeders":2,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":791772,"groupName":"The Beatles - Get Back","torrentId":1722119,"tags":["coffee.table.book","e.book","get.back","the.beatles"],"category":"E-Books","fileCount":1,"groupTime":"2021-11-14 16:44:07","size":695661477,"snatches":4,"seeders":5,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":789352,"groupName":"Richard Balls - A Furious Devotion: The Authorised Story of Shane MacGowan","torrentId":1715319,"tags":["biography","non.fiction"],"category":"Audiobooks","fileCount":18,"groupTime":"2021-11-08 05:59:02","size":310938842,"snatches":2,"seeders":3,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":787652,"groupName":"Anthony Kiedis - Scar Tissue","torrentId":1711255,"tags":["alternative.rock","funk","funk.rock","heavy.metal","pop","punk","rap.rock","rock"],"category":"Audiobooks","fileCount":16,"groupTime":"2021-11-03 22:21:36","size":1284435297,"snatches":3,"seeders":2,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":787649,"groupName":"Brian Head Welch - With My Eyes Wide Open","torrentId":1711245,"tags":["alternative.rock","industrial","metal","nu.metal","rock"],"category":"Audiobooks","fileCount":1,"groupTime":"2021-11-03 21:39:25","size":267474599,"snatches":2,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":785242,"groupName":"Oliver Sacks - Musicophilia: Tales of Music and the Brain","torrentId":1705248,"tags":["music","selfhelp"],"category":"Audiobooks","fileCount":13,"groupTime":"2021-10-28 07:59:05","size":320341327,"snatches":2,"seeders":2,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":785240,"groupName":"Alex Ross -  The Rest Is Noise: Listening to the Twentieth Century","torrentId":1705246,"tags":["literarycriticism","musiccriticism","musicintroduction","pulitzerprizewinner"],"category":"Audiobooks","fileCount":456,"groupTime":"2021-10-28 07:50:55","size":669348978,"snatches":1,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":785097,"groupName":"Dave Grohl - The Storyteller: Tales of Life and Music","torrentId":1704926,"tags":["autobiography","biography"],"category":"Audiobooks","fileCount":28,"groupTime":"2021-10-27 21:22:24","size":306305100,"snatches":6,"seeders":5,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":785009,"groupName":"Ian MacDonald - Revolution in the Head: The Beatles Records and the Sixties","torrentId":1704759,"tags":["historycriticism","humorentertainment","pop.culture"],"category":"Audiobooks","fileCount":1,"groupTime":"2021-10-27 16:02:21","size":340462404,"snatches":1,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":784189,"groupName":"Colin McAllister - Learning to Play Guitar: Chords, Scales, and Solos","torrentId":1702706,"tags":["greatcourses","audiobook","guitar"],"category":"Audiobooks","fileCount":25,"groupTime":"2021-10-24 18:44:45","size":727845392,"snatches":0,"seeders":0,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":783879,"groupName":"Sophia Chang - The Baddest Bitch in the Room: A Memoir","torrentId":1702148,"tags":["epub","hip.hop","memoir","musician"],"category":"E-Books","fileCount":1,"groupTime":"2021-10-23 20:56:14","size":7661026,"snatches":2,"seeders":2,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":783874,"groupName":"Joe Layden and Noel Monk - Runnin' with the Devil: A Backstage Pass to the Wild Times, Loud Rock, and the Down and Dirty Truth Behind the Making of Van Halen","torrentId":1702142,"tags":["arts","biographies","composers","literature","memoirs","musicians","epub"],"category":"E-Books","fileCount":1,"groupTime":"2021-10-23 20:45:01","size":11795886,"snatches":1,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":783834,"groupName":"Rob Kenner - The Marathon Don't Stop: The Life and Times of Nipsey Hussle","torrentId":1702051,"tags":["epub","hip.hop","music"],"category":"E-Books","fileCount":1,"groupTime":"2021-10-23 17:54:54","size":3957338,"snatches":1,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":783832,"groupName":"Jessica Simpson - Open Book: A Memoir","torrentId":1702049,"tags":["audiobook","autobiography"],"category":"Audiobooks","fileCount":40,"groupTime":"2021-10-23 17:39:34","size":327920810,"snatches":0,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":783828,"groupName":"Brandi Carlile - Broken Horses: A Memoir","torrentId":1702042,"tags":["audiobook","biography"],"category":"Audiobooks","fileCount":3,"groupTime":"2021-10-23 17:31:38","size":304774190,"snatches":3,"seeders":2,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":783801,"groupName":"Elton John - Me: Elton John Official Autobiography","torrentId":1701996,"tags":["audible","autobiography","me","music","audiobook"],"category":"Audiobooks","fileCount":19,"groupTime":"2021-10-23 16:05:23","size":259534126,"snatches":1,"seeders":0,"leechers":1,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":777867,"groupName":"Alicia Keys - More Myself: A Journey","torrentId":1689693,"tags":["inaudible.rip","autobiography","individual.composer.musician","music"],"category":"Audiobooks","fileCount":1,"groupTime":"2021-10-04 22:47:50","size":286276993,"snatches":1,"seeders":0,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":773024,"groupName":"David Katz - People Funny Boy: The Genius of Lee \"Scratch\" Perry","torrentId":1679997,"tags":["biography","dub","jamaican","reggae"],"category":"E-Books","fileCount":1,"groupTime":"2021-09-18 08:11:07","size":5482719,"snatches":3,"seeders":3,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":771040,"groupName":"Learn & Master Piano","torrentId":1675782,"tags":["piano"],"category":"E-Books","fileCount":1,"groupTime":"2021-09-11 13:39:14","size":29237931,"snatches":2,"seeders":3,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":770703,"groupName":"Stephen Witt - How Music Got Free","torrentId":1675124,"tags":["copyright","epub","file.sharing","non.fiction","piracy"],"category":"Audiobooks","fileCount":23,"groupTime":"2021-09-10 07:55:04","size":263999845,"snatches":2,"seeders":2,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":767485,"groupName":"Aebersold - Jazz Play-A-Longs [CD MP3 + PDF eBooks] 113-pack","torrentId":1669343,"tags":["backing.tracks","e.learning","ebook","elearning","jazz","music","play.along","tutorial","e.book","jam.tracks"],"category":"E-Books","fileCount":1659,"groupTime":"2021-08-31 12:49:04","size":10962188606,"snatches":4,"seeders":3,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":767219,"groupName":"Jarett Kobek - Do Every Thing Wrong!: XXXTentacion Against the World","torrentId":1668851,"tags":["biography","e.book","music","music.criticism","music.history"],"category":"E-Books","fileCount":2,"groupTime":"2021-08-30 17:03:14","size":1459122,"snatches":1,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":763223,"groupName":"Lori Burns, Serge Lacasse, J. Peter Burkholder - The Pop Palimpsest: Intertextuality in Recorded Popular Music","torrentId":1661106,"tags":["e.book","music","pop"],"category":"E-Books","fileCount":1,"groupTime":"2021-08-19 08:35:46","size":3132147,"snatches":1,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":763222,"groupName":"Marek Korczynski - Songs of the factory: pop music, culture, and resistance","torrentId":1661105,"tags":["e.book","music","pop"],"category":"E-Books","fileCount":1,"groupTime":"2021-08-19 08:35:14","size":693905,"snatches":0,"seeders":0,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":763221,"groupName":"Michael Fuhr - Globalization and Popular Music in South Korea: Sounding Out K-Pop","torrentId":1661104,"tags":["e.book","k.pop","pop"],"category":"E-Books","fileCount":1,"groupTime":"2021-08-19 08:34:40","size":2729527,"snatches":2,"seeders":2,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":763220,"groupName":"Peter Blake - Pop art and popular music jukebox modernism","torrentId":1661103,"tags":["e.book","music","pop"],"category":"E-Books","fileCount":1,"groupTime":"2021-08-19 08:34:04","size":6635551,"snatches":0,"seeders":0,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":763197,"groupName":"Tim Smith, Michael Tilson Thomas - The NPR curious listener's guide to classical music","torrentId":1661057,"tags":["classical","music"],"category":"E-Books","fileCount":1,"groupTime":"2021-08-19 07:00:13","size":9932308,"snatches":1,"seeders":2,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":763196,"groupName":"Nicole V. Gagn\u00e9 - Historical Dictionary of Modern and Contemporary Classical Music","torrentId":1661056,"tags":["classical","dictonary","e.book"],"category":"E-Books","fileCount":1,"groupTime":"2021-08-19 06:59:36","size":2397657,"snatches":0,"seeders":0,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":763195,"groupName":"Anna Beer - Sounds and Sweet Airs: The Forgotten Women of Classical Music","torrentId":1661055,"tags":["classical","music"],"category":"E-Books","fileCount":1,"groupTime":"2021-08-19 06:58:56","size":9441869,"snatches":0,"seeders":0,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":763194,"groupName":"Kent Nagano , Inge Kloepfer - Classical Music: Expect the Unexpected","torrentId":1661054,"tags":["classical","music"],"category":"E-Books","fileCount":1,"groupTime":"2021-08-19 06:58:23","size":1729412,"snatches":0,"seeders":0,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":763193,"groupName":"Patrick Laude, Barry McDonald - Music of the Sky: An Anthology of Spiritual Poetry (Spiritual Classics)","torrentId":1661053,"tags":["e.book"],"category":"E-Books","fileCount":1,"groupTime":"2021-08-19 06:57:49","size":1684278,"snatches":0,"seeders":0,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":763192,"groupName":"Janaki Bakle - Two Men and Music: Nationalism in the Making of an Indian Classical Tradition","torrentId":1661052,"tags":["indian"],"category":"E-Books","fileCount":1,"groupTime":"2021-08-19 06:57:17","size":2841926,"snatches":1,"seeders":1,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false},{"groupId":763190,"groupName":"Marianna Ritchey - Composing Capital: Classical Music in the Neoliberal Era","torrentId":1661048,"tags":["classical","music"],"category":"E-Books","fileCount":1,"groupTime":"2021-08-19 06:50:55","size":845585,"snatches":0,"seeders":0,"leechers":0,"isFreeleech":false,"isNeutralLeech":false,"isPersonalFreeleech":false,"canUseToken":true,"hasSnatched":false}]},"info":{"source":"Orpheus","version":1}};FeedUrl: https://orpheus.network/ajax.php?action=browse&order_by=time&order_way=desc&filter_cat[3]=1&filter_cat[4]=1&filter_cat[7]=1

2022-01-11 12:27:39.3|Info|ReleaseSearchService|Searching indexer(s): [kickasstorrents.ws] for Term: [George Michael Very Best], Offset: 0, Limit: 100, Categories: [3000]
2022-01-11 12:27:39.4|Info|Cardigann|Adding request: https://kick4ss.com/usearch/George%20Michael%20Very%20Best/?field=time_add&sorder=desc
2022-01-11 12:27:39.3|Info|ReleaseSearchService|Searching indexer(s): [SDBits] for Term: [George Michael Very Best], Offset: 0, Limit: 100, Categories: [3000, 100006]
2022-01-11 12:27:39.3|Info|ReleaseSearchService|Searching indexer(s): [Orpheus] for Term: [], Offset: 0, Limit: 100, Categories: [3000, 3030]
2022-01-11 12:27:39.4|Info|Cardigann|Adding request: https://sdbits.org/browse.php?c6=1&c4=1&search=George+Michael+Very+Best&incldead=1&descriptions=0&imdbgt=0&imdblt=10&imdb=&sort=seeders&d=DESC
2022-01-11 12:27:39.3|Info|ReleaseSearchService|Searching indexer(s): [HDBits] for Term: [George Michael Very Best], Offset: 0, Limit: 100, Categories: [3000]
2022-01-11 12:27:39.4|Info|ReleaseSearchService|Searching indexer(s): [MyAnonamouse] for Term: [George Michael Very Best], Offset: 0, Limit: 100, Categories: [3000, 3030]
2022-01-11 12:27:39.4|Info|ReleaseSearchService|Searching indexer(s): [FileList] for Term: [George Michael Very Best], Offset: 0, Limit: 100, Categories: [3000, 3040, 100005, 100011]
2022-01-11 12:27:39.4|Info|ReleaseSearchService|Searching indexer(s): [1337x] for Term: [], Offset: 0, Limit: 100, Categories: [3000, 3010, 3040, 3050, 3030]
2022-01-11 12:27:39.4|Info|Cardigann|Adding request: https://1337x.to/sort-search/Very%20Best/time/desc/1/
2022-01-11 12:27:39.4|Info|ReleaseSearchService|Searching indexer(s): [Torlock] for Term: [George Michael Very Best], Offset: 0, Limit: 100, Categories: [3000, 3030]
2022-01-11 12:27:39.4|Info|Cardigann|Adding request: https://www.torlock.com/music/torrents/george-michael-very-best.html?sort=added&order=desc
2022-01-11 12:27:39.4|Info|ReleaseSearchService|Searching indexer(s): [The Pirate Bay] for Term: [George Michael Very Best], Offset: 0, Limit: 100, Categories: [3000, 3030, 3040, 3050]
2022-01-11 12:27:39.4|Info|Cardigann|Adding request: https://apibay.org/q.php?q=george.michael.very.best&cat=100%2C101%2C102%2C103%2C104%2C199
2022-01-11 12:27:40.1|Info|Cardigann|Adding request: https://1337x.to/sort-search/Very%20Best/time/desc/2/
2022-01-11 12:27:41.8|Info|Cardigann|Adding request: https://1337x.to/sort-search/Very%20Best/time/desc/3/

[AB#2352](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/2352)

**Comment by bakerboy448** [comment]
It's not a healthcheck error. It's an unhandled time response from the indexer.

The healthcheck is accurate.

**Comment by jmerhar** [comment]
> It's not a healthcheck error. It's an unhandled time response from the indexer.
> 
> The healthcheck is accurate.

I'm not sure I understand the difference. Should I rephrase the issue?

**Comment by bakerboy448** [comment]
Edited the title; so a slight rephrasing of the issue wouldn't hurt

The healthcheck is accurate - the indexer is not returning a valid response.  

Trying to convert a datetime to an integer is what needs to be fixed

**Comment by ryanmurray** [comment]
ill note i get invalid torrent when trying to grab any release through lidarr coming through prowlarr

**Comment by jmerhar** [comment]
> ill note i get invalid torrent when trying to grab any release through lidarr coming through prowlarr

This is probably a separate issue, since searching and grabbing torrents works fine for me, despite the health check failure.

**Comment by bakerboy448** [comment]
Waiting on a response from Orpheus as to whether their deviation from the Gazelle standard is a bug or a permanent change.

**Comment by ryanmurray** [comment]
> > ill note i get invalid torrent when trying to grab any release through lidarr coming through prowlarr
> 
> This is probably a separate issue, since searching and grabbing torrents works fine for me, despite the health check failure.

I figured it out setting freeleech if you have none causes this

**Comment by ta264** [comment]
Orpheus seems to have fixed this

**Comment by bakerboy448** [comment]
Orpheus seems to have this occurring again
https://discord.com/channels/767843854736949299/767843854824374281/969311165236797552

**Comment by bakerboy448** [comment]
Appears to be isolated to book searches

https://gist.github.com/bakerboy448/1c9571830b1574ab8188a4d71c836442

https://discord.com/channels/767843854736949299/767843854824374281/969325407285436466
